install/kubernetes/cilium/values.yaml

```diff
@@ -2114,7 +2114,7 @@ readinessProbe:
 kubeProxyReplacementHealthzBindAddr: ""
 l2NeighDiscovery:
   # -- Enable L2 neighbor discovery in the agent
-  enabled: true
+  enabled: false
   # -- Override the agent's default neighbor resolution refresh period.
   refreshPeriod: "30s"
```

Comment:
Isn't this removed also? Please also document the removal in the upgrade notes.

Reply:
Yes, it was removed in an earlier commit. I forgot to remove it from the values. Will fix. Change is already noted in the upgrade nodes.